zeth_contracts/contracts/MixerBase.sol

```diff
@@ -116,6 +132,70 @@ contract MixerBase is BaseMerkleTree, ERC223ReceivingContract
         num_inputs_out = NUM_INPUTS;
     }
 
+    /// Permitted dispatchers may call this entry point if they have verified
+    /// the associated proof. This is technically part of the
+    /// IZecaleApplication interface, see
+    /// https://github.com/clearmatics/zecale
+    function dispatch(
+        uint256 nested_vk_hash,
+        uint256[] memory nested_inputs,
+        bytes memory nested_parameters
+    )
+        public
+        payable
+    {
+        // Sanity / permission checkcheck
+        require(
+            msg.sender == _permitted_dispatcher,
+            "dispatcher not permitted");
```

Comment:
Let's break before closing the `require`

Reply:
I realised that both params actually fit on a single line.  Changed.